integration/testutil.go

```diff
@@ -220,7 +221,7 @@ func NewEnv(t *testing.T) *Env {
 		t.Fatalf("Dial(%v) = %v", addr, err)
 	}
 	cli := pb.NewKeyTransparencyServiceClient(cc)
-	client := grpcc.New(mapID, cli, vrfPub, verifier, fake.NewFakeTrillianClient())
+	client := grpcc.New(mapID, cli, vrfPub, verifier, fake.NewFakeTrillianLogVerifier())
```

Comment:
Not any more.  Previously, clients needed a `TrillianClient` to connect to the log server itself. Now clients just need to verify the output of Trillian.  Meanwhile, the keyserver itself needs the `TrillianClient` in order to proxy those requests now.